inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/CacheCluster.java

```diff
@@ -21,13 +21,15 @@
  * CacheCluster
  */
 public class CacheCluster {
```

Comment:
Please move those classes to the manager-common module, thanks.

Reply:
Removed